bin/xbps-remove/main.c

```diff
@@ -158,7 +173,7 @@ remove_pkg(struct xbps_handle *xhp, const char *pkgname, bool recursive)
 int
 main(int argc, char **argv)
 {
-	const char *shortopts = "C:c:dFfhnOoRr:vVy";
+	const char *shortopts = "C:c:dFfhH:nOk:oRr:vVy";
```

Comment:
Why did you add new stuff?

Reply:
Basically, **keep_cache** parameter represents the number of previous versions of the packages that we want to keep in the cache making easier an eventual downgrade. Please ignore 'H' option, you only consider the new 'k' option. Copy and paste from my library in which i have more stuff.

Reply:
The idea is to have more control when we remove the obsolete packages from cache or local repository via "keep" option in both xbps-remove and xbps-rindex command. Currently, we always remove all previous versions of packages in both cache and local repository.